### PR TITLE
no-if-six.py*

### PR DESCRIPTION
**Pull request by drowoseque**
## Description
`if six.PY2` / `if six.PY3` removed from base code

## Motivation and Context
https://github.com/spotify/luigi/issues/2830

### CODE REVIEW

**Comment by drowoseque** [comment]
@honnix 
@dlstadther 
ptal

**Comment by honnix** [comment]
Sorry for the conflicts.

**Comment by drowoseque** [comment]
@honnix 
i've resolved them

File: test/worker_test.py [hunk 1]
```diff
@@ -1483,26 +1482,6 @@ def complete(self):
 
 class MultipleWorkersTest(unittest.TestCase):
 
-    @unittest.skip('Always skip. There are many intermittent failures')
-    # This pass under python3 when run as `nosetests test/worker_test.py`
-    # but not as `nosetests test`. Probably some side effect on previous tests
-    @unittest.skipIf(six.PY3, 'This test fail on python3 when run with tox.')
```
Comment (honnix) [inline test/worker_test.py]: This is not strictly python3 related. It's flaky test that we can decide to keep or nor out side of this PR. I think it's fine we just remove this line and the comments above but still skip it.

**Comment by drowoseque** [comment]
@honnix 
@dlstadther 
now ci is passed as well as coverage checks
can we go on?

**Pull request merged**